owl/EASE-OBJ.owl

```diff
@@ -26,6 +26,46 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#affects -->
+
+    <owl:ObjectProperty rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#affects">
+        <rdfs:subPropertyOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#hasConstituent"/>
```

Comment:
note: I have not seen a property that would be a better match. e.g. 'is related to concept' is only concept-to-concept.

Reply:
but what should be the super-property? I haven't seen a good fit

Reply:
Quality taxonomy has moved to concept branch.